src/Sdk/RSWebApi/Contracts/CompleteJobRequest.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using GitHub.DistributedTask.WebApi;
+
+namespace GitHub.Actions.RunService.WebApi
+{
+    [DataContract]
+    public class CompleteJobRequest
+    {
+        [DataMember(Name = "planId", EmitDefaultValue = false)]
+        public Guid PlanID { get; set; }
+        
+        [DataMember(Name = "jobId", EmitDefaultValue = false)]
+        public Guid JobID { get; set; }
+        
+        [DataMember(Name = "conclusion")]
```

Comment:
do you need `EmitDefaultValue = false`?

Reply:
no, I thnk default needs to be there, like if its `0` for example

Reply:
same as https://github.com/actions/runner/blob/9a228e52e9fb22857052cbb8a63e8f61f98f56a4/src/Sdk/DTWebApi/WebApi/JobEvent.cs#L179